packages/discovery-react-components/src/components/SearchFacets/components/__tests__/FieldFacets.test.tsx

```diff
@@ -485,7 +485,7 @@ describe('FieldFacetsComponent', () => {
 
     describe('when 2 selections are made in the same category', () => {
       beforeEach(async () => {
-        setupData = setup({ filter: 'author:"ABMN Staff"|author:"News Staff"' });
+        setupData = setup({ filter: 'author:ABMN Staff|author:News Staff' });
```

Comment:
So is this fine? There's no need for quotes or parentheses around "ABMN Staff"?

Reply:
This is a value before it's processed. We add the parens when we create the query to send in the payload. If you look around the tests, the filters being sent into the setups are all pre-processed versions, and then we expect processed versions.